pages/docs/faqs.mdx

```diff
@@ -6,19 +6,19 @@ It is completely free to use LinkFree.
 
 ## Is it all Open Source?
 
-Yes, LinkFree is 100% Open Source. You can use our hosted version, or you can host it yourself in [Single User Mode](docs/single-user-mode).
+Yes, LinkFree is 100% Open Source. You can use our hosted version, or you can host it yourself in [Single User Mode](advanced/single-user-mode).
```

Comment:
```suggestion
Yes, LinkFree is 100% Open Source. You can use our hosted version, or you can host it yourself in [Single User Mode](/advanced/single-user-mode).
```

Reply:
Not sure if it should have `docs` on it? I tried that on Gitpod and it lead to `docs/docs`.
As you're already on the docs page, it'll redirect you to `advanced/single-user-mode`.

I also saw other docs pages do this (without the `docs`) and work.

Reply:
It shouldn't matter where someone is, if all links start with `/`